TUTORIALS.md

```diff
@@ -1,98 +1,86 @@
+    registry.cn-beijing.aliyuncs.com/adlik/model-compiler:7.0.0.11_10.0 bash
+    ```
 
-## Infer model
+4. Compile the model: convert the .h5 file to TensorFlow serving model
 
-**First**, Prepare inference client code (see [image_client.py](adlik_serving/clients/python/image_client.py)) and data
-sets required for inference.
+   In the docker container:
 
-**Second**, infer the model.
+    ```sh
+    cd /home/john/model/
 
-```sh
-python3 client.py --batch-size=1 image
-```
+    python3 compile_model.py
+    ```
 
-## Test the serving engine performance
+   After that there is a serving model in the “model_repos” directory
 
-### Automatic test the serving engine performance
+### Deploy tensorflow serving service
 
-**First**, use [Git](https://git-scm.com/download) clone the Adlik code or download zip.
+In the `Adlik/examples/keras_model` directory, run the command:
 
-**Second**, follow the step in [benchmark](benchmark/README.md), you can get the test result.
+1. Pull the serving docker image
 
-**Third**, if you want to test the performance of the new serving engine which provided by yourself, you can write a new
-serving Dockerfile based on the [existing serving Dockerfile](benchmark/tests/docker_test), then following the step 2.
+    ```sh
+    docker pull registry.cn-beijing.aliyuncs.com/adlik/serving/tensorflow-cpu:latest
+    ```
 
-### Manually test serving engine performance
+2. Run the image
 
-1. [Install and build Adlik](#Install-and-build-Adlik).
+    ```sh
+    docker run -d -p 8500:8500 -v $PWD/model_repos:/srv/adlik-serving registry.cn-beijing.aliyuncs.com/adlik/serving/tensorflow-cpu:latest
+    ```
 
-2. [Optimize models](#Optimize-model) based on demand.
+### Infer model
 
-3. [Compile model](#Compile-model).
+In the `Adlik/examples/keras_model` directory, run the command:
 
-4. [Deploy serving service](#Deploy-serving-service). In this way, you need to redirect log files, by:
+1. Install the `Adlik serving` package
 
     ```sh
-    ./adlik_serving --model_base_path={model_repos_dir} --grpc_port={grpc_port} --http_port={http_port} >> log_path 2>&1
-    ```
+    wget https://github.com/Adlik/Adlik/releases/download/v0.2.0/adlik_serving_api-0.0.0-py2.py3-none-any.whl
 
-5. Run a client and do inference:
-
-   ```sh
-   cd {Adlik_root_dir}/benchmark/test/client
-   python3 client.py --batch-size=128 path_image
-   ```
+    python3 -m pip install adlik_serving_api-0.0.0-py2.py3-none-any.whl
+    ```
 
-   Save the log of client.
+2. Run a client and do inference:
 
-6. Analyze inference results
+    ```sh
+    python3 mnist_client.py data/0.png
+    ```
 
-   ```sh
-   cd {Adlik_root_dir}/benchmark/src
-   python3 test_result.py path_client_log path_serving_log batch_size model_name runtime
-   ```
+   The result as follows:
 
-   Then you can get the performance analysis results of the serving.
+    ```text
+    Image: 'data/0.png', result: [{'idx': 0, 'score': 0.9999995231628418, 'label': ''}]
+    ```
```

Comment:
Add something like:
You can also convert PyTorch, Keras, TensorFlow models to OpenVINO and TensorRT serving model and deploy these serving models easily with our images.